api/specs/director/openapi.yaml

```diff
@@ -128,6 +128,41 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorEnveloped"
 
+  /services/{service_key}/{service_version}:labels:
```

Comment:
should be:
```/services/{service_key}/{service_version}/labels:```
it's a resource, not a custom call

Reply:
done